src/site/xdoc/beginning_development.xml

```diff
@@ -17,7 +17,7 @@
     </section>
     <section name="Before Development">
       <p>
-        1. Ensure that Git, Java JDK >= 11 until JDK 17, maven >= 3.6.3 are installed.<br/>
+        1. Ensure that Git, Java JDK >= 17, maven >= 3.6.3 are installed.<br/>
```

Comment:
as we are here and changing same line, please revove "maven >= 3.6.3 are" as we 
3.9.6 
https://github.com/checkstyle/checkstyle/blob/26124ef95311c0c6f2f2105bb812b23a57b0b8bd/.mvn/wrapper/maven-wrapper.properties#L17-L19

and this version is not required at all as required mvn will be installed by any run of build by "mvnw"